test/distributed/algorithms/quantization/quantization_test.py

```diff
@@ -140,3 +170,29 @@ def _test_all_to_all(
             for t1, t2 in zip(out_tensors, expected_tensors):
                 self.assertEqual(t1, t2)
         self._barrier()
+
+    def _test_all_to_all_single_unequal_split_helper(
+        self, group, group_id, rank, cuda=False, rank_to_GPU=None, dtype=torch.float, qtype=None
+    ):
+        if group_id is not None:
+            size = len(group)
+            in_splits = [i + 1 for i in group]
+            out_splits = [rank + 1 for _ in group]
+            in_tensor = torch.ones([sum(in_splits), size], dtype=dtype) * rank
+            out_tensor = torch.ones([(rank + 1) * size, size], dtype=dtype)
+            expected_tensor = torch.cat(
+                [torch.ones([rank + 1, size], dtype=dtype) * i for i in group]
+            )
+            if cuda:
+                in_tensor = in_tensor.cuda(rank_to_GPU[rank][0])
+                expected_tensor = expected_tensor.cuda(rank_to_GPU[rank][0])
+                out_tensor = out_tensor.cuda(rank_to_GPU[rank][0])
+            if(qtype is not None):
+                quantize_alltoall_single = quant.auto_quantize(dist.all_to_all_single, qtype, quant_loss=None)
```

Comment:
qtype must be none for our tests cases right? I think we should make `qtype` not an optional argument as we just want to test the quantization part, if `qtype=None`, the tests should already been covered by `distributed_test.py`